freqtrade/commands/arguments.py

```diff
@@ -12,7 +12,7 @@
 
 ARGS_COMMON = ["verbosity", "logfile", "version", "config", "datadir", "user_data_dir"]
 
-ARGS_STRATEGY = ["strategy", "strategy_path"]
+ARGS_STRATEGY = ["strategy", "strategy_path", "recursive_strategy_search"]
```

Comment:
This is what you were missing.

in general, i'd however think that this is mostly a "config" setting - which you set in your config and forget about.
you'll usually not want to run `--reqcursive-strategy-search` with every command - assuming that's your standard way to work.